bigtop-packages/src/common/hive/do-component-build

```diff
@@ -32,11 +32,21 @@ HIVE_MAVEN_OPTS=" -Dhbase.version=$HBASE_VERSION \
 # Include common Maven Deployment logic
 . $(dirname ${0})/maven_deploy.sh
 
+# workaround for BIGTOP-3548, as the protoc binary included in
+# com.github.os72::protoc doesn't work with legacy centos-7 libc
+# so we use protoc built by bigtop_toolchain to create a fake jar
+source /etc/os-release
+case ${ID}-${VERSION_ID} in
+    centos-7)
+        mvn install:install-file -DgroupId=com.github.os72 -DartifactId=protoc -Dversion=2.6.1-build3 \
+            -Dclassifier=linux-aarch_64 -Dpackaging=exe -Dfile=/usr/local/bin/protoc
```

Comment:
"linux-aarch_64" is hardcoded. This should be changed to reflect certain non-x86 platform.